ComponentKit/Core/CKComponentController.h

```diff
@@ -79,4 +79,14 @@
  */
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender;
 
+
+/**
+ Initializes a controller with the first generation of component. You should not directly initialize a controller,
+ they are initialized for you by the infrastructure.
+ */
+- (instancetype)initWithComponent:(CKComponent *)component NS_DESIGNATED_INITIALIZER;
```

Comment:
Honestly I think this is fine. There are other breaking changes in the roadmap (removing `CKComponentDataSource` in favor of `CKTransactionalComponentDataSource` being the most notable) so I'm not too worried about this one.